tests/helpers.py

```diff
@@ -90,3 +90,11 @@ def send(self, *args):
 
 
 _MockResponse = namedtuple('MockResponse', ['status_code', 'text', 'headers'])
+
+
+def assert_raises(exception_class, action):
+  try:
+    action()
+    raise AssertionError, "Should have raised an %s" % exception_class
```

Comment:
We'll also want to raise if a different exception is raised than what we expected.